Cargo.toml

```diff
@@ -17,23 +17,24 @@ include = ["/src", "LICENSE", "README.md", "CHANGELOG.md"]
 thiserror = "1.0"
 log = "0.4"
 imgui = "^0.12"
-ash = { version = ">=0.34, <=0.37", default-features = false, features = ["debug"] }
+ash = { version = "0.38", default-features = false, features = ["debug"] }
 ultraviolet = "0.9"
 
-gpu-allocator = { version = "0.26", default-features = false, features = ["vulkan"], optional = true }
+#gpu-allocator = { version = "0.26", default-features = false, features = ["vulkan"], optional = true }
+gpu-allocator = { git = "https://github.com/Traverse-Research/gpu-allocator.git", branch = "ash-0.38", default-features = false, features = ["vulkan"], optional = true }
 
-vk-mem = { version = "0.3", optional = true }
+vk-mem = { version = "0.4", optional = true }
 
 [features]
 dynamic-rendering = []
 
 [dev-dependencies]
 simple_logger = "5.0"
-winit = { version = "0.29", default-features = false, features = ["rwh_05", "x11", "wayland", "wayland-dlopen", "wayland-csd-adwaita"]}
+winit = { version = "0.29", default-features = false, features = ["rwh_06", "x11", "wayland", "wayland-dlopen", "wayland-csd-adwaita"]}
 imgui = { version = "^0.12", features = ["tables-api"] }
 imgui-winit-support = "^0.12"
-ash = { version = "0.37", default-features = false, features = ["debug", "linked"] }
-ash-window = "0.12"
-raw-window-handle = "0.5"
+ash = { version = "0.38", default-features = false, features = ["debug", "linked"] }
+ash-window = "0.13"
+raw-window-handle = "0.6"
```

Comment:
`winit` re-exports `raw-window-handle` as `winit::raw-window-handle`. 
Should we use the re-export ?

Reply:
went ahead and made the proposed change